### PR TITLE
Update mandelbulb & sdf files

### PR DESCRIPTION
**Pull request by kfahn22**
Add link to gist with example frag file to mandelbulb files
Add references to superShape & mandelbulb to sdf file

### CODE REVIEW

File: sdf/mandelbulbSDF.glsl [hunk 1]
```diff
@@ -3,7 +3,11 @@
 /*
 original_author: Kathy McGuiness
 description: Returns the mandelbulb SDF
+For more information about the mandlebulb, check [this article](https://en.wikipedia.org/wiki/Mandelbulb)
```
Comment (patriciogonzalezvivo) [inline sdf/mandelbulbSDF.glsl:6]: please respect the YAML structure of the comments. For multiple lines do:
```yaml
description: |
    Something blabla
    other line something bla bla
```

File: sdf/mandelbulbSDF.glsl [hunk 2]
```diff
@@ -3,7 +3,11 @@
...
+
 use: mandelbulbSDF(<vec2> st)
+Example:
+https://gist.github.com/kfahn22/4c29e6d2bdc33d639f315edaa934d287
```
Comment (patriciogonzalezvivo) [inline sdf/mandelbulbSDF.glsl:10]: Same here, respect the YAML structure of the comment header. `examples` field is not capitalize and examples are part of of a list.
```yaml
examples:
    - https://gist.githubusercontent.com/kfahn22/4c29e6d2bdc33d639f315edaa934d287/raw/0eb94683614995476fb63cfa99881e420f1c5be6/mandelbulb.frag
```

also the example needs to be a text page not html. Use the RAW link

File: sdf/mandelbulbSDF.hlsl [hunk 3]
```diff
@@ -3,7 +3,11 @@
 /*
 original_author: Kathy McGuiness
 description: Returns the mandelbulb SDF
+For more information about the mandlebulb, check [this article](https://en.wikipedia.org/wiki/Mandelbulb)
```
Comment (patriciogonzalezvivo) [inline sdf/mandelbulbSDF.hlsl:6]: same notes as above

**Review by patriciogonzalezvivo — CHANGES_REQUESTED** [review]

File: sdf/mandelbulbSDF.glsl (hunk 1 above)
Comment (kfahn22) [reply sdf/mandelbulbSDF.glsl]: I have updated the file.

File: sdf/mandelbulbSDF.glsl (hunk 2 above)
Comment (kfahn22) [reply sdf/mandelbulbSDF.glsl]: I have updated the file.

**Comment by kfahn22** [comment]
Sorry about mistakes, I don't have a lot of experience with YAM!  Please let me know if any additional changes are needed.

**Comment by patriciogonzalezvivo** [comment]
thank you!

**Pull request merged**